cmd/agent.go

```diff
@@ -433,6 +434,7 @@ func (ar *AgentRunner) runInstance() error {
 					Findings:     &res.Findings,
 					Risks:        &res.Risks,
 					Logs:         &res.Logs,
```

Comment:
```suggestion
					Logs:         &res.Logs,
					// TODO Use scheduler time here, rather than loop time. This is intentionally left at 1 minute for now. Once cron scheduling is in, we should change it. This TODO mainly serves as a signal to increase chances of not missing the update later
```